schema.sql

```diff
@@ -1,5 +1,12 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    /*id INT SERIAL PRIMARY KEY*/
+    id INT GENERATED BY DEFAULT AS IDENTITY,
+    name VARCHAR(255),
+    date_of_birth DATE,
+    escape_attempts INT,
+    neutered BOOLEAN,
+    weight_kg DECIMAL(6, 2),
+    PRIMARY KEY(id)
 );
```

Comment:
**[Optional]**

- [ ] It would be nice if you add screenshots in your pull request summary because it is a requirement of your project and it also makes your pull request professional :blush: